sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -388,6 +571,47 @@ internal List<ArraySegment<byte>> ConvertLockTokensToDeliveryTags(IEnumerable<Gu
             return lockedUntilUtc;
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        public override async Task<DateTime> RenewSessionLockAsync(string sessionId, CancellationToken cancellationToken = default)
```

Comment:
Can we remove the `sessionId` param and use the class property?

Reply:
Yes